src/vlogger/vlogger.cpp

```diff
@@ -216,58 +242,113 @@ void printf_backtrace(void)
 #pragma BullseyeCoverage on
 #endif
 
-////////////////////////////////////////////////////////////////////////////////
-// NOTE: this function matches 'bool xlio_log_set_cb_func(xlio_log_cb_t log_cb)' that
-// we gave customers; hence, you must not change our side without considering their side
-static xlio_log_cb_t xlio_log_get_cb_func()
+// room for color code termination and EOL
+#define VLOGGER_STR_TERMINATION_SIZE 6
+
+// for the extreme case where we have a failure before initializing doca logger
+static void output_before_doca_logger(vlog_levels_t log_level, const char *fmt, ...)
```

Comment:
lets review together

Reply:
Yup - let's talk. maybe it's not needed.